src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -159,6 +159,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help=" Allow the blob service to exhibit filesystem semantics. This property can be enabled only "
                    "when storage account kind is StorageV2.",
                    min_api='2018-02-01')
+        c.argument('encryption_key_type_for_table', arg_type=get_enum_type(['Account', 'Service']),
```

Comment:
Can we define a CLIArgumentType (encryption_key_type) for reuse instead of defining for both table and queue.

Reply:
It is a little different. In this way, I haven't use CLIArugmentType.

Reply:
Ok, in the future, if there is more places to use this arg_type, we can extract them for re-use.

---
In reply to: [365073751](https://github.com/Azure/azure-cli/pull/11706#discussion_r365073751) [](ancestors = 365073751)